test/controllers/invalid.company.status.controller.unit.ts

```diff
@@ -0,0 +1,49 @@
+jest.mock("../../src/services/company.profile.service");
+
+import mocks from "../mocks/all.middleware.mock";
+import request from "supertest";
+import app from "../../src/app";
+import { getCompanyProfile } from "../../src/services/company.profile.service";
+import { validCompanyProfile } from "../mocks/company.profile.mock";
+import { INVALID_COMPANY_STATUS_PATH, URL_QUERY_PARAM } from "../../src/types/page.urls";
+import { urlUtils } from "../../src/utils/url";
+
+const mockGetCompanyProfile = getCompanyProfile as jest.Mock;
+
+const STOP_PAGE_TITLE_COMPANY_STATUS = "You cannot use this service - Company Status";
+const SERVICE_UNAVAILABLE_TEXT = "Sorry, the service is unavailable";
+
+describe("Invalid company status controller tests", () => {
+
+  beforeEach(() => {
+    mocks.mockAuthenticationMiddleware.mockClear();
+    mocks.mockServiceAvailabilityMiddleware.mockClear();
+    mocks.mockSessionMiddleware.mockClear();
+    mockGetCompanyProfile.mockClear();
+  });
+
+  describe("get tests", () => {
```

Comment:
Could we rename this so it's a bit clearer to "get function tests" or "test get function"?